src/SWP/Bundle/ContentBundle/Model/ImageRendition.php

```diff
@@ -118,4 +93,9 @@ public function setName(string $name): void
     {
         $this->name = $name;
     }
+
+    public function isConvertedToWebp(): bool
+    {
+        return $this->getImage()->haveVariant('webp');
```

Comment:
it should be `hasVariant`. You can assign `webp` to a const in `ImageInterface` as it's used in more places.